composer.json

```diff
@@ -22,10 +22,14 @@
         "source": "https://github.com/portphp/doctrine",
         "docs": "https://portphp.readthedocs.org"
     },
+    "minimum-stability": "dev",    
+    "prefer-stable": true,
     "require": {
         "php": ">=5.4.0",
         "portphp/portphp": "1.0.x-dev",
-        "doctrine/orm": "~2.4"
+        "doctrine/orm": "~2.4",
```

Comment:
At the end all implementations should go into suggest/require-dev and we should only require doctrine/common to make it work with the object manager.

This would also allow us to write spec and integration tests separately.

@ddeboer thoughts?